tobac/merge_split.py

```diff
@@ -22,6 +44,8 @@ def merge_split_MEST(TRACK, dxy, distance=None, frame_len=5):
         The x/y grid spacing of the data.
         Should be in meters.
 
+    dz : float, optional
```

Comment:
In the rest of the 3D code, we support both constant vertical grid spacing and user-specified varying vertical grid spacing. We can either require constant vertical grid spacing for this code (less ideal) and check that the user didn't pass in an array-like, or update the code to support both (if it doesn't already). I'm happy enough with either outcome.